startcloudwatchlogs.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Get the instance region and inject it in the conf
+REGION=$(ec2metadata --availability-zone | head -c-2)
```

Comment:
is ec2metadata present on the machine, you may need to install it, think its cloud-utils maybe

Reply:
according to [this](https://medium.com/@jbirdvegas/query-ec2-metadata-on-ubuntu-dc6cfbe17940) it is cloud utils, will add

Reply:
it wasn't, have now added